core/ui/gui_util.h

```diff
@@ -280,15 +280,15 @@ static inline bool iconButton(const char *icon, const std::string& label, const
 {
 	ImguiStyleVar _{ImGuiStyleVar_ButtonTextAlign, ImVec2(0.f, 0.5f)};	// left aligned
 	std::string s(5 + label.size(), '\0');
-	s.resize(sprintf(s.data(), "%s  %s", icon, label.c_str()));
+	s.resize(snprintf(s.data(), sizeof(s.data()), "%s  %s", icon, label.c_str()));
```

Comment:
`std::string::data()` returns a char pointer so `sizeof` won't work in this case